app/uk/gov/hmrc/timetopayproxy/models/CreatePlanRequest.scala

```diff
@@ -52,7 +52,16 @@ object PlanToCreatePlan {
 }
 
 final case class PaymentInformation(paymentMethod: PaymentMethod, paymentReference: Option[PaymentReference]) {
-  require(!paymentReference.forall(_.value.trim().isEmpty()), "paymentReference should not be empty")
+  if (paymentMethod == PaymentMethod.DirectDebit)
```

Comment:
Hi even if we will use require for now (assuming that another ticket should be created to convert these into Validation classes and getting rid of requires alltogether, we should not use if's in my humble opinion. instead we can convert them into statements like these

    require(
      (paymentMethod != PaymentMethod.DirectDebit) || (paymentMethod == PaymentMethod.DirectDebit) && !paymentReference.forall(_.value.trim().isEmpty()),
      "Direct Debit should always have payment reference"
    )
but better crosscheck on your side too. thanks